### ISSUE TITLE
Several models for MI 

### ISSUE THREAD

**Issue by javadbayazi** [XiaoxiWei/NeurIPS_BEETL#2]
based on `start_kits/4.LeaderboardLabelGeneration.ipynb` > `checkpoint = torch.load('D:\\motor_imagery\\MImodelinLeaderboard\\'+'cnn_model_MI_'+pilotname+'.pth')` , it seems we can have an specific pretrained model for each subject. If not, what does 'pilotname' in the checkpoint means?

**Comment by XiaoxiWei** [comment]
Hi, thanks for the question. Once you use only the data provided, and the pre-trained model is not trained with data other than the data sets in the competition, it's fine. We do not limit if you are using one model for all subjects. You can either use one model for all or have a unique model for one subject (while integrating some transfer learning techniques). Besides, this file is only for demonstration of loading a model you trained, then use that model to generate labels, but not 'a pre-train finetune' example.

**Comment by XiaoxiWei** [comment]
Closed issue.
